pegasus/sites.v3/code.org/public/educate/weblab-test.haml

```diff
@@ -1,8 +1,2 @@
----
-title: Web Lab Test
-theme: responsive
----
-
-%script{src: webpack_asset_path('js/code.org/public/educate/weblab-test.js')}
-
-#weblab-test
+:ruby
+  redirect CDO.studio_url('/weblab/test', CDO.default_scheme)
```

Comment:
just out of curiosity, why redirect instead of removing this route? were users accessing it already?

Reply:
Yeah, Hannah has been giving it out to folks for troubleshooting purposes. I asked about removing it, but redirecting was recommended in standup today.